src/main/java/uk/gov/ons/ctp/response/collection/exercise/distribution/SampleUnitDistributor.java

```
@@ -305,6 +338,5 @@ private CollectionExercise collectionExerciseTransitionState(CollectionExercise
     } catch (CTPException ex) {
       log.error("Collection Exercise state transition failed", ex);
```

I know it's not part of this PR, but this "log and fail silently" is at the root of so many of our issues in the CI pipeline and acceptance tests

---

This `collectionExerciseTransistion` method hasn't been touch by this PR so will leave as is
It's a mess though so we should look at improving it going forward